draft-ietf-tls-esni.md

```diff
@@ -850,21 +850,21 @@ If the client offers the "ech_is_inner" extension ({{is-inner}})
 in addition to the "encrypted_client_hello" extension, the server MUST abort
 with an "illegal_parameter" alert.
 
-First, the server collects a set of candidate ECHConfigs. This set is
```

Comment:
Would it be possible to keep the "set of candidate $X" phrasing?

The phrase "candidate ECHConfigList" sounds to me like the whole ECHConfigList is a candidate, not that each element is a candidate. 

Perhaps

> First, the server collects a set of candidate ECHConfig values.

Reply:
Sure, that works equally well for me!